puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/tests/testForeman.groovy

```diff
@@ -0,0 +1,135 @@
+node('fast') {
+
+    stage('Setup Git Repos') {
+
+        git url: "https://github.com/theforeman/foreman", branch: "develop"
+
+    }
+
+    stage("Setup RVM") {
+
+        withRVM([
+            'rvm gemset empty --force',
+            'gem update --no-ri --no-rdoc',
+            'gem install bundler --no-ri --no-rdoc'
+        ])
+
+    }
+
+    stage('Configure Environment') {
+
+        try {
+            sh "cp config/settings.yaml.example config/settings.yaml"
+            sh "sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.yaml"
+            sh "sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml"
+
+            sh "cp $HOME/postgresql.db.yaml config/database.yml"
+
+            sh "sed -i \"s/database:.*/database: ${gemset()}-test/\" config/database.yml"
+            sh """
+cat <<EOT >> config/database.yml
+development:
+adapter: postgresql
+database: ${gemset()}-development
+username: foreman
+password: foreman
+host: localhost
+template: template0
+
+production:
+adapter: postgresql
+database: ${gemset()}-development
+username: foreman
+password: foreman
+host: localhost
+template: template0
+EOT
+            """
+        } catch (all) {
+            cleanup()
+        }
+
+    }
+
+    stage('Configure Database') {
+
+        try {
+
+            withRVM([
+                'bundle update',
+                'bundle exec rake db:drop db:create -q',
+                'bundle exec rake db:migrate -q'
```

Comment:
One thing that came up when talking to @ShimShtein earlier today, can we also run `bundle exec rake db:seed` here? Quick test suggests yes.

Reply:
I added it to the old jobs here: https://github.com/theforeman/foreman-infra/pull/325